File: InsertionSort/InsertionSort.py
def insertionSort(value):
    
    for i in range(len(value)):
        j = i
        while (j > 0 and value[j] <= value[j-1]):
            value[j], value[j-1] = value[j-1], value[j]
            j -= 1
    return value

a = [1,3,7,9,5]
b = [5,5,7,3,0,2,1,52,10,6,37]

print insertionSort(a)
print insertionSort(b)

File: InsertionSort/InsertionSort.java
import java.util.Arrays;

class InsertionSort{
    void sort(int[] arr){
        for (int i = 0; i < arr.length; i++) {
            int j = i;
            while (j > 0 && arr[j] <= arr[j-1]){
                int aux = arr[j];
                arr[j] = arr[j-1];
                arr[j-1] = aux;
                j -= 1;
                
            }
        }
    }
    void printArray(int[] arr){
        System.out.println(Arrays.toString(arr));
    }

    public static void main(String[] args) {
        int[] a = {9,5,6,7,3,1};
        int[] b = {456,789,123};

        InsertionSort is = new InsertionSort();
        is.sort(a);
        is.sort(b);
        is.printArray(a);
        is.printArray(b);
    }

}